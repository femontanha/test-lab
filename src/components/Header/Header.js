import React from 'react';
import PropTypes from 'prop-types';
import styles from './header.css';

const propTypes = {
  title: PropTypes.string.isRequired,
};

const Header = ({ title }) => {
  return (
    <header className={ styles.header }>
      <h1 className={ styles.title }>{ title }</h1>
    </header>
  );
};

Header.propTypes = propTypes;

export default Header;
