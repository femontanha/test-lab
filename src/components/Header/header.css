.header {
  background-color: var(--primary-bg-color);
}

.title {
  color: var(--primary-title-color);
  font-size: 32px;
  margin: 0;
  padding: 30px 0;
  text-align: center;
}
