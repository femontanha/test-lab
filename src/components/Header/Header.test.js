import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('render Header component', () => {
  const component = renderer.create(
    <Header title={ 'React Fake Posts' } />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
