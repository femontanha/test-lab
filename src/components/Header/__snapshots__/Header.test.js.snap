// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header component 1`] = `
<header
  className={undefined}
>
  <h1
    className={undefined}
  >
    React Fake Posts
  </h1>
</header>
`;
